src/InputForm.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputFormType {
+  setAddTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const InputForm: React.FC<InputFormType> = ({ setAddTodoIsClicked }) => {
```

Comment:
It's better to use a more specific name for the prop, like 'onAddTodo' instead of 'setAddTodoIsClicked'.

Reply:
I renamed this one, but I assume you want me to rename the rest of the similar props (((